python/doc/examples/reliability_sensitivity/design_of_experiments/plot_smolyak_merge.py

```diff
@@ -113,13 +112,9 @@ def computeNumberOfSmolyakNodes(level, epsilon_a, epsilon_r):
     index += 1
 palette = ot.Drawable.BuildDefaultPalette(number_of_epsilons)
 graph.setColors(palette)
-graph.setLegendPosition("topright")
-otv.View(
-    graph,
-    figure_kw={"figsize": (4.0, 3.0)},
```

Comment:
That is not the purpose of this commit. The figure size setting must be kept as in the original code: only the legend setting must be updated here.